hack/generator/pkg/codegen/pipeline_delete_generated_code.go

```diff
@@ -0,0 +1,162 @@
+package codegen
```

Comment:
header check failed for a few of these files

Reply:
Fixed.